src/main/java/lotto/machine/WinningSystem.java

```diff
@@ -0,0 +1,44 @@
+package lotto.machine;
+
+import lotto.repository.Memory;
+import lotto.constant.Constant;
+
+import java.util.List;
+
+public class WinningSystem {
+    private final Memory memory;
+
+    public WinningSystem(Memory memory) {
+        this.memory = memory;
+    }
+
+    public void checkWinning(List<Lotto> tickets, List<Integer> winningNumbers, int bonusNumber) {
```

Comment:
매개변수 명이 일정하지 않은 것 같습니다. 의도된 경우가 아니라면 이름은 대체로 동일하게 해주는 것이 좋습니다.
```suggestion
    public void checkWinning(List<Lotto> lottoTickets, List<Integer> winningNumbers, int bonusNumber) {
```